### DISCUSSION TITLE
klipper was working for an hour but now moonraker wont connect to klipper.

### DISCUSSION THREAD

**Discussion by unrelated-productions** [bassamanator/Sovol-SV06-firmware#146] [Q&A]
I got Klipper running through the mainsail interface and did some of the configuring steps, but had to leave for a bit before finishing up. I shutdown my raspberry pi 3 using "Sudo Shutdown -now". The next time I turned on the raspberry pi and mainsail I got the message 

"Moonraker can't connect to Klipper! Please check if the Klipper service is running and klippy_uds_address is correctly configured in the moonraker.conf." 

I've uninstalled and reinstalled klipper, moonraker, and mainsail through KIAUH, checked the serial id but nothing has changed.

I've attached the moonraker log, but not the klipper log as it is just a 404 error code.
[moonraker(3).log](https://github.com/bassamanator/Sovol-SV06-firmware/files/14233602/moonraker.3.log)

Thank you for your time and help!

**Comment by bassamanator** [comment]
Check the Adjust Configuration with MCU Path section, try both methods.

**Reply by unrelated-productions** [reply]
if that makes no change what do you suggest I try next?

**Comment by bassamanator** [comment]
Check what groups the user is part of: `groups $USER`.
